src/tzdata/build.jl

```diff
@@ -38,25 +38,57 @@ function build(
         end
     end
 
-    archive = joinpath(archive_dir, "tzdata$version.tar.gz")
-
-    # Avoid downloading a tzdata archive if we already have a local copy
-    if version == "latest" || !isfile(archive)
-        @info "Downloading $version tzdata"
-        archive = tzdata_download(version, archive_dir)
-
+    @static if VERSION >= v"1.4"
+        now_utc = now(Dates.UTC)
+        # todo: I don't have latest here and it should not be used as latest
         if version == "latest"
-            m = match(TZDATA_VERSION_REGEX, basename(archive))
+            m = match(TZDATA_VERSION_REGEX, "tzdata$version.tar.gz")
             if m !== nothing
                 version = m.match
                 @info "Latest tzdata is $version"
+                artifact_dir = @artifact_str "tzdata_$version"
+                version = tzdata_version_dir(artifact_dir)
+                set_latest(version, now_utc)
+            else
+                archive = tzdata_download(version, archive_dir)
+                if !isempty(tz_source_dir)
+                    @info "Extracting $version tzdata archive"
+                    extract(archive, tz_source_dir, setdiff(regions, CUSTOM_REGIONS), verbose=verbose)
+                end
             end
         end
```

Comment:
I'll just point out that if `version == "latest"` then the regex will never match ("latest" will never be matched by the regex) which results in the `m !== nothing` check never actually getting called. Same problem as pointed out in https://github.com/JuliaTime/TimeZones.jl/pull/270/files#r454385743.

I wouldn't worry about this as I can fix it in a follow up PR.

Reply:
Ok, I see, thanks for pointing it out.